packages/aws-cdk-lib/aws-stepfunctions/lib/aokencryptionconfiguration.ts

```diff
@@ -0,0 +1,12 @@
+import { EncryptionConfiguration } from './encryptionconfiguration';
+
+const AWS_OWNED_KEY = 'AWS_OWNED_KEY';
```

Comment:
Can you create an enum for this and change the type of `type: string` to `type: whateverEnum`.

Reply:
Spoke with paulhcsun offline, they said it's fine to use const.